richContracts.json

```diff
@@ -101,28 +101,27 @@
     },
     "functions": [
       {
-        "title": "Vote on a proposal",
+        "title": "Vote For proposal",
         "functionName": "vote",
         "params": [
           {
             "type": "bytes32",
             "component": "address",
```

Comment:
I just discovered something in testing, this should be `component: "string"` and same for the against action